components/hab/src/command/studio/enter.rs

```diff
@@ -43,6 +45,22 @@ fn set_env_var_from_config(env_var: &str, config_val: Option<String>, sensitive:
     }
 }
 
+fn cache_ssl_cert_file(cert_file: &str, cert_cache_dir: &Path) {
+    let cert_path = Path::new(&cert_file);
```

Comment:
I've found myself getting confused when reading the logic of this function, and I think a few naming changes may make it easier to follow:
1. `cert_file` → `file_to_cache` or `src_cert`, etc.
1. `cert_cache_dir` → `dest_dir`
1. When converting `cert_file` of type `&str` → `Path`, re-bind the same name rather than introduce `cert_path`* (this is allowed and idiomatic) or make the function take a `Path` as input rather than `&str`.

These are just suggestions; not required changes

\* Since rust has such strong typing, in general I find it useful to put only the purpose of the data in the variable name and leave out words that describe the type.